app/components/forms/wizard/custom-form-rich-text.ts

```diff
@@ -0,0 +1,248 @@
+import { action, computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+import DS from 'ember-data';
+import { tracked } from '@glimmer/tracking';
+import { translateLanguages } from 'open-event-frontend/utils/dictionary/translate-language';
+import { A } from '@ember/array';
+
+interface CustomForm {
+  fieldIdentifier: string,
+  name: string,
+  type: string,
+  min: number,
+  max: number,
+  formIdentifier: string,
+  translations: Translate[],
+  mainLanguage: string
+}
+
+interface SubForm {
+  ignoreLanguages:string[],
+  name: string,
+  languages: object,
+  selectedLang: string,
+  isDeleted: boolean,
+  id: any,
+  form_id: string
+}
+
+interface Translate {
+  name: string,
+  code: string,
+  isDeleted: boolean,
+  id: string,
+  form_id: string,
+  language_code: string
+}
+
+function getIdentifier(name: string, fields: CustomForm[]): string {
+  const fieldIdentifiers = new Set(fields.map(field => field.fieldIdentifier));
+  let identifier = slugify(name, '_');
+  while (fieldIdentifiers.has(identifier)) {
+    identifier += '_';
+  }
+
+  return identifier;
+}
+
+interface Args {
+  field: CustomForm | null,
+  customForms: CustomForm[],
+  form: string,
+  event: any,
+  formIdentifier: string | '',
+  mainLanguage: string | 'en',
+  min: number | 0,
+  max: number | 10,
+  onSave: (() => void) | null
+}
+
+export default class CustomFormInput extends Component<Args> {
+  @tracked
+  name = '';
+
+  @tracked
+  type = 'text';
+
+  @service
+  store!: DS.Store;
+
+  @tracked
+  min = 0;
+
+  @tracked
+  max = 10;
+
+  @tracked
+  mainLanguage = 'en'
+
+  @tracked
+  selectedLanguage:string[] = [this.mainLanguage]
+
+  @tracked
+  subForm:SubForm[] = A([]);
+
+  translations:Translate[] = [];
+
+  @action
+  updated(): void {
+    this.subForm.clear()
+    if (this.args.field) {
+      this.name = this.args.field.name;
+      this.type = this.args.field.type;
+      this.min = this.args.field.min;
+      this.max = this.args.field.max;
+      this.mainLanguage = this.args.field.mainLanguage;
+      this.translations = this.args.field.translations;
+      this.selectedLanguage.clear();
+      this.selectedLanguage.pushObject(this.mainLanguage)
+      const selectedLanguage = this.translations?.map(trans => trans.code || trans.language_code);
+      this.selectedLanguage.pushObjects(selectedLanguage)
+      this.translations?.forEach((trans: Translate) => {
+        const { name, code, language_code, form_id, isDeleted, id } = trans;
+        if (isDeleted || (name && (code || language_code))) {
+          this.subForm.pushObject({
+            id,
+            form_id,
+            name,
+            languages       : translateLanguages,
+            ignoreLanguages : this.selectedLanguage,
+            selectedLang    : code || language_code,
+            isDeleted
+          })
+        }
+      })
+    } else {
+      this.name = '';
+      this.min = 0;
+      this.max = 10;
```

Comment:
Create a variable with MIN_VAL, MAX_VAL instead of directly assigning 0 & 10

Reply:
this is just default value so creating variable to set default is redundant as value is only used once